### PR TITLE
1.0.0 doc changes

### PR DESCRIPTION
**Pull request by cs-abhishek-shukla**
- Added screenshots in usage
- Elaborated README.md with possible use cases
- Added an SS with custom data in setup.md
- Fixed typos

### CODE REVIEW

File: docs/setup.md [hunk 1]
```diff
@@ -10,23 +10,25 @@
...
-| Relationship Nodes                         | Select the picklist to render relationship nodes.                                                          |
-| External Source JSON                       | Enter JSON data to render sources which generates an events.                                                                                       |
+A Sankey chart needs a source node and a target node. It then represents the flow from the source node to the target node using paths known as links. Following table helps customize the **SOC Overview Sankey** widget:
+
+| Fields                | Description                                                                         |
+|-----------------------|-------------------------------------------------------------------------------------|
+| Title                 | Specify a title for the SOC Overview Sankey widget as it appears on the dashboard.  |
+| Resource              | Select the module whose Sankey representation is to be displayed.                   |
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: suggestion: whose Sankey representation is to be displayed, i.e. FortiSOAR module

File: README.md [hunk 2]
```diff
@@ -12,7 +12,33 @@
...
+6. **User Activity and Access:**
+   - Use the Sankey chart to visualize user activity and access patterns within the FortiSOAR&trade; environment. Nodes can represent users or roles, and links can show their interactions with different systems or processes.
+
+7. **Custom Metrics and KPIs:**
+   - Customize the Sankey chart to display key performance indicators (KPIs) and metrics relevant to the FortiSOAR&trade;'s goals, such as incident resolution time, automation efficiency, or alert response times.
+
+8. **Real-time Monitoring:**
+   - Integrate the Sankey chart into a real-time monitoring dashboard within FortiSOAR to provide FortiSOAR&trade; analysts with a dynamic and up-to-date view of the security operations flow.
+
+When implementing a Sankey chart widget in FortiSOAR, ensure that it supports dynamic updates and real-time data feeds to provide a current and accurate representation of the FortiSOAR&trade; workflow. Additionally, customize the chart to meet the specific needs and processes of your FortiSOAR&trade; environment.
```
Comment (cs-suyog-jinturkar) [inline README.md]: ensure that it supports dynamic updates and real-time data feeds to provide a current and accurate representation of the FortiSOAR&trade; workflow ---> What is expected by the user here?

Comment (cs-suyog-jinturkar) [reply README.md]: We might need to add more details or example here for the user

File: docs/setup.md [hunk 3]
```diff
@@ -10,23 +10,25 @@
...
+| Filter Criteria       | Define the filter criteria using which to filter the data retrieved by this widget. |
+| Source Nodes          | Select the field to render source nodes.                                            |
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md]: any field or picklist field?

File: docs/setup.md [hunk 4]
```diff
@@ -10,23 +10,25 @@
...
+| External Source JSON  | Enter JSON data to render sources which generates an events.                        |
 
 ## External Source JSON format
-Here JSON object requires 2 array properties, Nodes and Links.
 
-Sample JSON format:
+Here JSON object requires 2 array properties, **`Nodes`** and **`Links`**. Value in **`Nodes`** act as a *`Source`* node for the module selected in the **Resource** field.
+
+Following is a sample JSON that can be entered in the **External Source JSON** field.
+
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md:40]: as per me, Data mentioned in the json is not same as data in the screenshot. Please correct, if needed.

File: docs/setup.md [hunk 5]
```diff
@@ -10,23 +10,25 @@
...
 ```JSON
```
Comment (cs-suyog-jinturkar) [inline docs/setup.md:41]: @cs-vikas-shelar please add more details around input json and its usage here.

File: docs/usage.md [hunk 6]
```diff
@@ -3,51 +3,82 @@
...
 
 You can add this view to a dashboard, or report.
 
-## Features
+## Visualize Relationship Data for a Preset Interval
 
-- Visualize relationship data as per duration. User can select from standard durations provided as part of widget like 7 Days, 1 Day, and 1 Hour. 
+You can select from following options:
 
+- 7 Days
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: last 7 days, last 1 day, etc.

File: docs/usage.md [hunk 7]
```diff
@@ -3,51 +3,82 @@
...
+4. Select the module, whose data is to be represented as a Sankey chart as per relationship, in the **Resource** field.
 
-5. Define the filter criteria using which to filter the data retrieved by this widget (Optional).
+    ![](./res/sankey-edit-03.png)
+
+5. Define the filter criteria using which to filter the data retrieved by this widget (Optional), in the **Filter Criteria** field. To know more about using the filter criteria, refer to the [Nested Filter](https://docs.fortinet.com/document/fortisoar/7.4.3/user-guide/207943/dashboards-templates-and-widgets#Nested-Filters) section of the FortiSOAR&trade; user guide.
+
+    ![](./res/sankey-edit-04.png)
 
 6. Select the text field to be grouped for source nodes, in the **Source Nodes** field.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md]: text field or picklist?

File: docs/usage.md [hunk 8]
```diff
@@ -3,51 +3,82 @@
...
 
-7. Select the picklist field to group data for target nodes, in the **Target Nodes** field.
+    ![](./res/sankey-edit-05.png)
+
+7. Select the field to group data for target nodes, in the **Target Nodes** field.
+
+    ![](./res/sankey-edit-06.png)
 
 8. Select the relationship module, to group data to visualize as per relation, in the **Relationship Resource** field.
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md:54]: We should describe this point better by giving more details around, what is relationship module means here?

File: docs/usage.md [hunk 9]
```diff
@@ -3,51 +3,82 @@
...
 
+    ![](./res/sankey-edit-08.png)
+
 10. Input JSON data to render outer source nodes and linking with modules source, in the **External Source JSON** field.
 
+    ![](./res/sankey-edit-09.png)
+
 ### SOC Overview Sankey Widget - View
 
 The following image displays a **SOC Overview Sankey** widget based on an example where you might want to view the data as per severity of related incidents of alerts group by source and severity:
```
Comment (cs-suyog-jinturkar) [inline docs/usage.md:68]: We should also render and illustrate one more example here taken from your overview section, say alert --> type --> severity --> escalated/not escalated or you can pick other example as well here

**Review by cs-suyog-jinturkar — COMMENTED** [review]
Added review comments

File: docs/setup.md (hunk 1 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md]: Fixed.

File: README.md (hunk 2 above)
Comment (cs-abhishek-shukla) [reply README.md]: Removed the line... the updates are real-time by default

File: docs/setup.md (hunk 3 above)
Comment (cs-abhishek-shukla) [reply docs/setup.md]: Fixed.

File: docs/usage.md (hunk 7 above)
Comment (cs-abhishek-shukla) [reply docs/usage.md]: fields of type text listed in the picklist

File: docs/setup.md [hunk 10]
```diff
@@ -10,47 +10,149 @@
...
+| Title                 | Specify a title for the SOC Overview Sankey widget as it appears on the dashboard.                                                             |
+| Resource              | Select the FortiSOAR&trade; module whose Sankey representation is to be displayed. For example, **_Alerts_**.                                  |
+| Filter Criteria       | Define the filter criteria using which to filter the data retrieved by this widget.                                                            |
+| Source Nodes          | Select the a field, within the **_Resource_** module, to render source nodes. Only the fields of type **_text_** are listed.                   |
+| Target Nodes          | Select the picklist, within the **_Resource_** module, to render target nodes. Only the fields of type **_picklist_** are listed.              |
+| Relationship Resource | Select the FortiSOAR&trade; module, which has a relationship to the module selected in the **Source Node** field. For example, **_Incidents_** |
```
Comment (cs-vikas-shelar) [inline docs/setup.md]: to the module selected in the **Resource** field.

**Review by cs-vikas-shelar — COMMENTED** [review]
The changes look good to me. 1 minor change is required for field detail.

**Review by cs-vikas-shelar — APPROVED** [review]
Looks good to me.

**Pull request merged**